modules/core/integration_test.go

```diff
@@ -0,0 +1,95 @@
+package ibc_test
+
+import (
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+	ibcmock "github.com/cosmos/ibc-go/v8/testing/mock"
+)
+
+// If packet receipts are pruned, it is possible to double spend via a
```

Comment:
```suggestion
// If packet receipts are pruned, it should not be possible to double spend via a
```

Reply:
i understood colin's comment to mean that double spending works if the proof is submitted, that's why we added the check against the packet recv seq start in addition to the proof verification? but im not super opinionated, i think it makes sense this way as well.

Reply:
yea exactly, I can update the comment to be more specific

Reply:
https://github.com/cosmos/ibc-go/pull/5651/commits/f23573484920ac0974af4780f49145b9423a3819 https://github.com/cosmos/ibc-go/pull/5651/commits/7d5f6e85fad217a1567828756a1738947698a7a1